tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -143,6 +143,64 @@ install_cached_component() {
 	mv "${component_tarball_name}" "${component_tarball_path}"
 }
 
+# We've to add a different cached function here as for using the shim-v2 caching
+# we have to rely and check some artefacts coming from the cc-rootfs-image and the
+# cc-tdx-rootfs-image jobs.
+install_cached_cc_shim_v2() {
+	local component="${1}"
+	local jenkins_build_url="${2}"
+	local current_version="${3}"
+	local current_image_version="${4}"
+	local component_tarball_name="${5}"
+	local component_tarball_path="${6}"
+	local root_hash_vanilla="${repo_root_dir}/tools/osbuilder/root_hash_vanilla.txt"
+	local root_hash_tdx="${repo_root_dir}/tools/osbuilder/root_hash_tdx.txt"
+
+	local rootfs_image_cached_root_hash="${jenkins_url}/job/kata-containers-2.0-rootfs-image-cc-$(uname -m)/${cached_artifacts_path}/root_hash_vanilla.txt"
+	local tdx_rootfs_image_cached_root_hash="${jenkins_url}/job/kata-containers-2.0-rootfs-image-tdx-cc-$(uname -m)/${cached_artifacts_path}/root_hash_tdx.txt"
+
```

Comment:
Nit: I'm not sure if the double new line is needed

Reply:
You're right, if something has to be changed I will also update this one, thanks!